components/sup/src/manager/mod.rs

```diff
@@ -277,12 +280,13 @@ impl Manager {
             events_group: cfg.eventsrv_group,
             launcher: launcher,
             services: services,
-            watcher: SpecWatcher::run(&fs_cfg.specs_path)?,
+            spec_watcher: SpecWatcher::run(&fs_cfg.specs_path)?,
             fs_cfg: Arc::new(fs_cfg),
             organization: cfg.organization,
             service_states: HashMap::new(),
             sys: Arc::new(sys),
             peer_watcher: peer_watcher,
+            user_config_watcher: UserConfigWatcher::new(),
```

Comment:
If you changed the order of members in `Manager` struct, it would be good to reorder the initializations here too.